requirements.txt

```diff
@@ -7,3 +7,4 @@ pandas
 PyGithub
 requests
 tabulate
+xonsh
```

Comment:
Neat! I didn't realize it's available as a PyPI package. Here is a link for future us: https://pypi.org/project/xonsh/.